Formula/swi-prolog.rb

```diff
@@ -24,14 +25,17 @@ class SwiProlog < Formula
   depends_on "pkg-config" => :build
   depends_on "berkeley-db"
   depends_on "gmp"
-  depends_on "jpeg"
```

Comment:
Didn't see any usage in current bottles. From source tarball, it looks like a dependency for X support, which we have been disabling.
```console
❯ rg 'find.*JPEG'
packages/xpce/CMakeLists.txt
24:  find_package(JPEG)
33:    find_package(JPEG)
63:        message("  Could not find JPEG")
```